diffrax/_brownian/base.py

```diff
@@ -8,7 +8,7 @@
 from .._path import AbstractPath
 
 
-class AbstractBrownianPath(AbstractPath):
+class AbstractBrownianPath(AbstractPath[Union[PyTree[Array], LevyVal]]):
```

Comment:
Note that this means that `AbstractBrownianPath` can no longer be subscripted. I think you probably mean something like
```python
_Control = TypeVar("_Control", bound=Union[PyTree[Array], LevyVal])

class AbstractBrownianPath(AbstractPath[_Control]):
```
to preserve the possibility of subscripting its subclasses.